### DISCUSSION TITLE
How to automate Thank You Messages to Buyers

### DISCUSSION THREAD

**Discussion by fahrettinozturk** [amzn/selling-partner-api-models#3645] [General]
Hello,

I was checking for the messaging API to see if we have an option to send "thank you messages" to the buyers through API. I see that we have only 3 options to send messages in the API.

- Remove Negative Feedback
- Create unexpected problem
- Send Invoice

It looks like there's not any option other than those three provided through API. I wanted to clarify that if I'm missing anything. With that, it's impossible to manage manually through Seller Central thousands of orders and messages. 

Here is the documentation for the messaging API:
https://developer-docs.amazon.com/sp-api/docs/messaging-api-v1-reference

**Comment by stefnats** [comment]
With every order, there comes a `BuyerEmail` which you can send emails to. This email is related to the specific order and is a specialized `@amazon.com` email address just for this order.
I would suggest that you get this email address either via reports or Orders API and then send an email there, i think the Messaging API is the wrong place for this.

Please also check the Amazon TOS if you don't violate them with what you want to do or it could backfire pretty badly.

**Reply by fahrettinozturk** [reply]
I'll try that. Thanks.
